### ISSUE TITLE
Cannot download, SSL cert failure

### ISSUE THREAD

**Issue by ResidentSpaceDragon** [datawhores/OF-Scraper#89]
Running the script goes as planned but all media is reported as skipped and exits, running debug throws error for every post. I dont know if its unrelated but clicking the of endpoint listed in the log goes to "Something went wrong" and logged me out of OF completely.
```
DEBUG    Media:XXX Post:YYY Attempting to download media                                         download.py:155
         1146x1968_33a35038855306151862d38882a64152 with
         https://cdn2.onlyfans.com/files/?Tag=2&Policy={hidden}&Signature={hidden}&Key-Pair-Id={hidden}
DEBUG    Media:XXX Post:YYY Downloading with normal downloader                                   download.py:156
DEBUG    Media:XXX Post:YYY  exception Cannot connect to host cdn2.onlyfans.com:443 ssl:True     download.py:148
         [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable
         to get local issuer certificate (_ssl.c:997)')]
DEBUG    Media:XXX Post:YYY  exception Traceback (most recent call last):                        download.py:149
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/connector.py", line 980, in
         _wrap_create_connection
             return await self._loop.create_connection(*args, **kwargs)  # type: ignore  # noqa
           File "/opt/anaconda/lib/pytho n3.10/asyncio/base_events.py", line 1097, in create_connection
             transport, protocol = await self._create_connection_transport(
           File "/opt/anaconda/lib/python3.10/asyncio/base_events.py", line 1127, in
         _create_connection_transport
             await waiter
           File "/opt/anaconda/lib/python3.10/asyncio/sslproto.py", line 534, in data_received
             ssldata, appdata = self._sslpipe.feed_ssldata(data)
           File "/opt/anaconda/lib/python3.10/asyncio/sslproto.py", line 188, in feed_ssldata
             self._sslobj.do_handshake()
           File "/opt/anaconda/lib/python3.10/ssl.py", line 975, in do_handshake
             self._sslobj.do_handshake()
         ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to
         get local issuer certificate (_ssl.c:997)

         The above exception was the direct cause of the following exception:

         Traceback (most recent call last):
           File "/home/user/.local/lib/python3.10/site-packages/ofscraper/utils/download.py", line 142, in
         download
             return await main_download_helper(c,ele,path,file_size_limit,username,model_id,progress)
           File "/home/user/.local/lib/python3.10/site-packages/ofscraper/utils/download.py", line 157, in
         main_download_helper
             async with c.get(url) as r:
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/client.py", line 1141, in
         __aenter__
             self._resp = await self._coro
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/client.py", line 536, in _request
             conn = await self._connector.connect(
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/connector.py", line 540, in connect
             proto = await self._create_connection(req, traces, timeout)
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/connector.py", line 901, in
         _create_connection
             _, proto = await self._create_direct_connection(req, traces, timeout)
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/connector.py", line 1206, in
         _create_direct_connection
             raise last_exc
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/connector.py", line 1175, in
         _create_direct_connection
             transp, proto = await self._wrap_create_connection(
           File "/home/user/.local/lib/python3.10/site-packages/aiohttp/connector.py", line 982, in
         _wrap_create_connection
             raise ClientConnectorCertificateError(req.connection_key, exc) from exc
         aiohttp.client_exceptions.ClientConnectorCertificateError: Cannot connect to host
         cdn2.onlyfans.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED]
         certificate verify failed: unable to get local issuer certificate (_ssl.c:997)')]
```

**Comment by datawhores** [comment]
Are you still having this issue

Also clicking any link will long you out.
This is a security measure

**Comment by ResidentSpaceDragon** [comment]
Yes,its still failing

**Comment by ResidentSpaceDragon** [comment]
Reverting to 2.4.5 solves this.
I updated my system certs and certifi however i was not able to get 2.5 latest to download, getting the same ssl errors.

**Comment by datawhores** [comment]
Hmm 
I pushing a update after this comment with the downloader reconfigured

**Comment by ResidentSpaceDragon** [comment]
Resolved in 2.6
